site/content/entry/yuzu-progress-report-jan-2024/index.md

```diff
@@ -0,0 +1,550 @@
+This leads to a noticeable frametime inconsistency when a lot of action is happening on screen. We hope AMD adds an option in the future to toggle the sensitivity of this behaviour.
+
+{{< youtube v-U7GJYrY64 >}} {{< youtube 8Cbov_uR2Dc >}}
+
+Native 30 FPS vs AFMF generating frames with a 30 FPS base.
+While a better result than Lossless Scaling at 30 FPS, artifacts are still noticeable in faster movements, especially on vegetation. Scene transitions are clear and there is less shimmering around the player.
+
+{{< youtube CG_e5yOnd9E >}} {{< youtube GfRjc9v0jls >}}
+Native 60 FPS vs AFMF generating frames with a 60 FPS base.
+And at 60 FPS the results are great, providing smooth gameplay to 120-180Hz displays.
+
+No performance graphs are included because your writer doesn’t consider perceived framerate as real performance, but to provide an example, `The Legend of Zelda: Tears of the Kingdom`, which produces solid 60 FPS in open world with a 5600X and an RTX 3060 Ti, can produce 170-180 “FPS” with AFMF enabled, and with little to no distortion visible.
+
+The games can be rendered with an NVIDIA or Intel GPU while still generating frames with AFMF, although the experience is slightly worse than just using the AMD card directly due to the added latency of transmitting the finished frames over PCIe.
+The only requisite for AFMF to work is to have the display connected to the compatible AMD GPU, any other GPU can do the actual rendering after that.
+Time to invest in an RX 6400? Smash must look amazing at 240 FPS.
+
+### Intel delivering on its promises
+
+As promised, Intel fixed [their crashes](https://github.com/IGCIT/Intel-GPU-Community-Issue-Tracker-IGCIT/issues/551) while building geometry shaders starting with driver version [31.0.101.5186/5234](https://www.intel.com/content/www/us/en/download/785597/intel-arc-iris-xe-graphics-windows.html).
+
+While this allows for games like `Xenoblade Chronicles 3` to finally get in-game on integrated GPUs and ARC dedicated cards, it seems like there are other areas where the driver needs to mature to produce proper rendering:
+
+{{< imgs
+	"./xc3.png| Intel moment (Xenoblade Chronicles 3)"
+  >}}
+
+On the flip side, `The Legend of Zelda: Tears of the Kingdom` improved stability with this driver fix, so Intel Windows specific rendering issues aside, the game should be quite playable now.
+
+### Qualcomm
+
+[K11MCH1](https://github.com/K11MCH1/AdrenoToolsDrivers/releases/tag/v24.1.0_R16)’s driver releases continue to flow out, improving performance and compatibility for the Adreno users. 
+We strongly recommend staying up-to-date.
+
+While there have been new releases of the proprietary driver, nothing of value has been added or fixed yet.
+
+Total Adreno moments count this month: 1.
+
+### Exynos and Mali
+
+Why together? Because we found some *interesting* limitations in the Exynos hardware that made us remember of the existential crisis Mali suffers.
```

Comment:
```suggestion
Why together? Because we found some *interesting* limitations in the Exynos hardware that reminds us of the existential crisis Mali suffers.
```